extensions/default/src/Panels/StudyBrowser/PanelStudyBrowser.tsx

```diff
@@ -102,6 +107,40 @@ function PanelStudyBrowser({
     ]
   );
 
+  const onClickThumbnailHandler = useCallback(
+    async displaySetInstanceUID => {
+      const customHandler = customizationService.getCustomization(
+        'studyBrowser.thumbnailClickCallback'
+      ) as CallbackCustomization;
+
+      if (!customHandler) {
+        return;
+      }
+
+      const setupArgs = {
+        activeViewportId,
+        commandsManager,
+        servicesManager,
+        isHangingProtocolLayout,
+        appConfig: extensionManager._appConfig,
```

Comment:
Does the app config actually need to be passed here?  Would rather pass the extension manager than extracting specific parts.